config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
 
 # Target score for pipeline.py:
 0
```

Comment:
you need to specify your expectations, otherwise tests will not run

Reply:
tests will not run otherwise. enable them asap or there might be last minute surprises